### PR TITLE
Minor fixes - update to OS/platform and services matrix and App Center bug tracker integration and four other articles

### PR DESCRIPTION
**Pull request by mahab339**
- In deploy Azure DevOps builds with App Center article, change (... for querying for the Store IDs using .) and (... to create a new connection.Paste the API key you copied...) to (... for querying for the Store IDs.) and (... to create a new connection. Paste...) respectively.

- In Intune company portal distribution article, change (... global admin of your company's Azure Active Directory global admin will need to go to... ) to (... Azure Active Directory will need to go to... ).

- In publish an iOS package to the App Store (Production) article, change (https://developer.apple.com/support/app-review/) to (https://developer.apple.com/app-store/review/). This fixes Apple review article broken URL.

- In OS/platform and services matrix article:
a - Add .NET MAUI Android and .NET MAUI iOS and their supported services in App Center: distribute, diagnostics, and analytics. 
References:
     - [App Center Distribute – MAUI and Xamarin In-app updates](https://learn.microsoft.com/en-us/appcenter/sdk/distribute/xamarin).
     - [App Center Crashes (MAUI and Xamarin)](https://learn.microsoft.com/en-us/appcenter/sdk/crashes/xamarin).
     - [App Center Analytics (MAUI and Xamarin)](https://learn.microsoft.com/en-us/appcenter/sdk/analytics/xamarin).
At least I am sure of diagnostics and analytics because I've used them. 

  b - Change (* - For UWP, HoloLens support hasn't been verified.) to (\\* - For UWP...) for literal render of (*).

- In App Center bug tracker integration article, change (\[work item types API](/rest/api/vsts/wit/work%20item%20types) for additional information.) to (... vsts/wit/) for additional...) as former one displays 404 page.

- In App Center webhooks article, change (For Distribute, the webhook will be triggered when you when a new version is released to a distribution group.) to (... will be triggered when a new version is released...).

### CODE REVIEW

File: docs/general/platform-service-matrix.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ The table below lists all the Operating System and Platforms and their supported
 | Xamarin - iOS          |  ✅   |  ✅  |     ✅       |   ✅       |    ✅     |
 | Xamarin - Android      |  ✅   |  ✅  |     ✅       |   ✅       |    ✅     |
 | Xamarin - macOS        |  ❌   |  ❌  |     ✅       |   ✅       |    ✅     |
+| .NET MAUI - Android    |  ❌   |  ❌  |     ✅       |   ✅       |    ✅     |
```
Comment (lucen-ms) [inline docs/general/platform-service-matrix.md]: We don't want to mention .NET MAUI at this moment, please remove.

File: docs/dashboard/bugtracker/index.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ App Center has bug tracker integration for the crashes service. Users can be qui
       1. Login with your Azure DevOps credentials and click **Accept** when prompted on app authorization.
       2. Select which Azure DevOps projects to integrate the bug tracker with and click **Next**.
       3. Toggle **On** for **Auto create ticket**, and fill in the fields for **Area** and **Default Payload**, and click **Add**:
-         - Default payload is an optional field to fill in for use in work items. For example, `{"System.IterationPath": "Area\\Iteration 1", "System.AssignedTo": "Fabrikam"}`. Refer to the [work item types API](/rest/api/vsts/wit/work%20item%20types) for additional information.
+         - Default payload is an optional field to fill in for use in work items. For example, `{"System.IterationPath": "Area\\Iteration 1", "System.AssignedTo": "Fabrikam"}`. Refer to the [work item types API](/rest/api/vsts/wit/) for additional information.
```
Comment (lucen-ms) [inline docs/dashboard/bugtracker/index.md]: The url should be `/rest/api/azure/devops/wit/work-item-types`

**Review by lucen-ms — CHANGES_REQUESTED** [review]

File: docs/general/platform-service-matrix.md (hunk 1 above)
Comment (mahab339) [reply docs/general/platform-service-matrix.md]: Done, please review.

File: docs/dashboard/bugtracker/index.md (hunk 2 above)
Comment (mahab339) [reply docs/dashboard/bugtracker/index.md]: Changed, please review.

**Review by lucen-ms — APPROVED** [review]

**Comment by lucen-ms** [comment]
@mahab339 thank you for your contribution!

**Pull request merged**